cpp/benchmarks/io/json/json.cpp

```diff
@@ -0,0 +1,161 @@
+/*
```

Comment:
should be separate the reader and writer benchmarks? During development, we usually care about a single component at a time.

Reply:
seperated.